pkg_defs/ska3-core-latest/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
    - jupyterlab
    - line_profiler
    - lxml
+   - mamba
    - matplotlib
```

Comment:
Probably also `pyplot`, since that is not a dependency of anything else.

Reply:
pyplot is not in ska3-core

Reply:
Or if Tom meant plotly, he already got that in in a separate PR.